labs/yocto-advanced-configuration/yocto-advanced-configuration.tex

```diff
@@ -172,7 +173,7 @@ \section{Going further}
 
 \begin{enumerate}
 
-\item Install a TFTP server (package \code{tftpd-hpa}) on your system. 
+\item Install a TFTP server (package \code{tftpd-hpa}) on your system.
 
```

Comment:
This is an unrelated change.

Reply:
I'm ready to take it anyway, as it's a minor cleanup one.